### ISSUE TITLE
is it possible that it recognizes only joining writing

### ISSUE THREAD

**Issue by f4r4zas** [MyScript/myscript-text-web#4]
Hey there is it possible that it will recognize only joining writing not simple writing

**Comment by MyScriptSupport** [comment]
Dear Faraz,
Thank you for your question.
Currently, our technogloy doesn't return with the gesture detection by itself.
The only way to proceed consits in checking the result, i.e. when 2 words are merged, you suppose a join gesture was done.
Best regards,
Olivier

**Comment by f4r4zas** [comment]
and can u recognize text using image?

**Comment by f4r4zas** [comment]
ok then can u aleast tell me is it possible to create aplhabet tracing app
for example i want to show a word " Testing  " dotted and on the canvas is it possible?

**Comment by MyScriptSupport** [comment]
Dear Faraz,

I am not sure to understand what you mean by "alphabet tracing app"?
Do you mean an "education application" in which the user would be asked to write a given word and you would use our solution to return the result and check it was correctly written?
if no, please explain what you expect.

Best regards,
Olivier

**Comment by f4r4zas** [comment]
yes that what i want but i want to give a word for example "Testing" and the word should be writting as dotted then user traces it and gets the result

**Comment by MyScriptSupport** [comment]
Dear Faras,

In that case, the solution is that you have an image in your canvas, and you write on top of it. Our TEXT solution will then return with the result.

Best regards,
Olivier

**Comment by MyScriptSupport** [comment]
Close #4
